exoctk/exoctk_app/form_validation.py

```diff
@@ -32,11 +38,10 @@ class FortneyModelForm(FlaskForm):
     clouds = SelectField('clouds', choices=[('0', 'Nothing'), ('ray10', 'Weak Rayleigh'), ('ray100', 'Medium Rayleigh'), ('ray1000', 'Strong Rayleigh'), ('flat10', 'Weak Cloud'), ('flat100', 'Medium Cloud'), ('flat1000', 'Strong Cloud')], validators=[InputRequired('A cloud model is required')])
 
     # Form submits
-    resolve_submit = SubmitField('Resolve Target')
```

Comment:
@hover2pi How many levels of nested Forms are we doing here? 
In this case -- because generic and fortney are going to be practically identical it might make sense to have them share a form. But we can discuss this in more detail when I'm actually done separating out the forward modeling.

Reply:
Yes! I thought this too. I made a general form for `forward_modeling` but didn't implement it. The functions you had to validate the inputs seemed fine so I figured we could wait until you did your stuff with the subpackage before implementing the WTForms solution.